docs/src/release-notes-js.md

```diff
@@ -6,6 +6,76 @@ toc_max_heading_level: 2
 
 import LiteYouTube from '@site/src/components/LiteYouTube';
 
+## Version 1.43
+
+### New APIs
+
+- Method [`method: BrowserContext.clearCookies`] now supports filters to remove only some cookies.
+
+  ```js
+  // Clear all cookies.
+  await context.clearCookies();
+  // New: clear cookies with a particular name.
+  await context.clearCookies({ name: 'session-id' });
+  // New: clear cookies for a particular domain.
+  await context.clearCookies({ domain: 'my-origin.com' });
+  ```
+
+- New trace mode `retain-on-first-failure`. In this mode, trace is recorded for the first run of each test, but not for retires. When test run failes, the trace file is retained, otherwise it is removed.
+
+  ```js title=playwright.config.ts
+  import { defineConfig } from '@playwright/test';
+
+  export default defineConfig({
+    use: {
+      trace: 'retain-on-first-failure',
+    },
+  });
+  ```
+
+- New property [`property: TestInfo.tags`] exposes test tags during test execution.
+
+- New method [`method: Locator.contentFrame`] converts a [Locator] object to a [FrameLocator]. Useful when you have a [Locator] object obtained somewhere, and later on would like to interact with the content inside the frame.
```

Comment:
```suggestion
- New method [`method: Locator.contentFrame`] converts a [Locator] object to a [FrameLocator]. It can be useful when you have a [Locator] object obtained somewhere, and later on would like to interact with the content inside the frame.
```